src/utils/typescript.ts

```diff
@@ -27,6 +27,9 @@ export const tryToRegisterTsNode = (): void => {
                 skipProject: JSON.parse(skipProjectRaw),
                 transpileOnly: JSON.parse(transpileOnlyRaw),
                 swc: JSON.parse(swcRaw),
+                compilerOptions: {
+                    allowJs: true,
+                },
```

Comment:
It gives ability to use `import/export` inside tests on javascript?

Reply:
If you use ts-node, yes